README.md

```diff
@@ -1,23 +1,118 @@
+Press Ctrl-C to interrupt
+Done checking log file disk usage. Usage is <1GB.
+
+started roslaunch server http://kuri-0001c:36850/
+
+. . .
+
+process[robot_state_publisher-2]: started with pid [19637]
+process[mobile_base-3]: started with pid [19638]
+process[depthscan-4]: started with pid [19639]
+process[madmux_daemon-5]: started with pid [19645]
+process[vision_bridge-6]: started with pid [19658]
+process[safety_controller-7]: started with pid [19665]
+process[head_controller-8]: started with pid [19668]
+process[chest_light_controller-9]: started with pid [19678]
+```
+
+At this point, you should observe that your Kuri robot's eyes open, the head 
+looks up, and the chest light begins to play a simple pattern.  The color of 
+the chest LED represents Kuri's battery level.  Green = 100%, Red = 20% or 
+lower.  Kuri should also back up when the bump shell is pressed.
+
+You can stop the kuri_edu software with ctrl-c
 
 ## Keyboard Teleop Demo
 
-To keyboard teleop a Kuri, run kuri_remote.launch or kuri_edu.launch on the robot:
+You can also drive Kuri like a remote control car using the 
+teleop_twist_keyboard node.  You may need to install this node on Kuri.
+
+```
+sudo apt-get update
+sudo apt-get install ros-indigo-teleop-twist-keyboard
+```
+
+Next, you'll need to start the kuri_edu software and the keyboard teleop node. 
+You can do this using two different terminals SSH'd to Kuri, or you can
+SSH to kuri and run `tmux` to multiplex your terminals, or you can run
+kuri_edu in the background like
+```
+roslaunch kuri_edu kuri_edu.launch &
+```
+
+If you launch kuri_edu in the background, you can stop it by bringing it to the
+forground with 
 ```
-roslaunch kuri_edu kuri_remote.launch
+fg
 ```
+and then stopping it with ctrl-c
 
-Then to use keyboard teleop (http://wiki.ros.org/teleop_twist_keyboard) on the robot, or on a remote system set up to communicate
-with the ROS master on Kuri:
+Once kuri_edu is running in another terminal or in the background, launch the
+keyboard teleop node:
 ```
 rosrun teleop_twist_keyboard teleop_twist_keyboard.py
 ```
 
-By default, kuri_remote disables all safety systems and allows the robot to be driven wherever the
-user wishes.  Be careful not to drive it down stairs!
+By default, kuri_edu disables all the cliff sesnors and allows Kuri to be 
+wherever you want.  Be careful not to drive it down stairs!
+
+### Troubleshooting
+If you get an error like:
+```
+mayfield@kuri-0001c:~$ sudo apt-get update
+
+. . .
+
+mayfield@kuri-0001c:~$ sudo apt-get install ros-indigo-teleop-twist-keyboard
+Reading package lists... Done
+Building dependency tree       
+Reading state information... Done
+E: Unable to locate package ros-indigo-teleop-twist-keyboard
+```
+
+You may need to add the ROS indigo debian sources to your robot.
+
+```
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
```

Comment:
Before this, you need to `sudo mkdir -p /etc/apt/sources.list.d`

Reply:
Added above